src/Compilers/CSharp/Portable/Parser/LanguageParser_InterpolatedString.cs

```diff
@@ -101,36 +126,242 @@ CodeAnalysis.Syntax.InternalSyntax.SyntaxList<InterpolatedStringContentSyntax> g
+                    content.Free();
+                }
+            }
+
+            CodeAnalysis.Syntax.InternalSyntax.SyntaxList<InterpolatedStringContentSyntax> getMultiLineRawContentWorker(
+                StringBuilder indentationWhitespace,
+                StringBuilder currentLineWhitespace,
+                StringBuilder content)
+            {
+                var builder = _pool.Allocate<InterpolatedStringContentSyntax>();
+
+                if (interpolations.Count == 0)
+                {
+                    // No interpolations.  Just grab the whole chunk of text and split it as appropriate.
+                    addContent(
+                        indentationWhitespace, currentLineWhitespace, content, builder, first: true, last: true,
+                        originalText[new Range(openQuoteRange.End, closeQuoteRange.Start)]);
+                }
+                else
+                {
+                    for (int i = 0; i < interpolations.Count; i++)
+                    {
+                        var interpolation = interpolations[i];
+
+                        // Add a token for text preceding the interpolation
+                        addContent(
+                            indentationWhitespace, currentLineWhitespace, content, builder, first: i == 0, last: false,
+                            originalText[new Range(
+                                i == 0 ? openQuoteRange.End : interpolations[i - 1].CloseBraceRange.End,
+                                interpolation.OpenBraceRange.Start)]);
+
+                        builder.Add(ParseInterpolation(this.Options, originalText, interpolation, kind));
                     }
+
+                    // Add a token for text following the last interpolation
+                    addContent(
+                        indentationWhitespace, currentLineWhitespace, content, builder, first: false, last: true,
+                        originalText[new Range(interpolations[^1].CloseBraceRange.End, closeQuoteRange.Start)]);
                 }
 
                 CodeAnalysis.Syntax.InternalSyntax.SyntaxList<InterpolatedStringContentSyntax> result = builder;
                 _pool.Free(builder);
                 return result;
             }
 
-            SyntaxToken getCloseQuote(Range openQuoteRange)
+            void addContent(
+                StringBuilder indentationWhitespace,
+                StringBuilder currentLineWhitespace,
+                StringBuilder content,
+                CodeAnalysis.Syntax.InternalSyntax.SyntaxListBuilder<InterpolatedStringContentSyntax> result,
+                bool first,
+                bool last,
+                string text)
+            {
+                if (text.Length == 0)
+                    return;
+
+                content.Clear();
+                var currentIndex = 0;
+
+                // If we're not the first content chunk, then we came after an interpolation.  In that case, we need to
```

Comment:
Phrasing?
Also grammatically, "we" are apparently:
- the content itself
- the thing consuming the content
- the reader/writer team maintaining the source code, presumably